maps/chronosphere/event_functions.lua

```diff
@@ -286,22 +286,35 @@ local mining_researches = {
 }
 
 function Public_event.mining_buffs(event)
+	if not global.difficulty_vote_value then global.difficulty_vote_value = 1 end
+	local difficulty = global.difficulty_vote_value
+	
+	local manual_mining_difficulty_scaling = 1.8
+	if difficulty < 1 then
+		manual_mining_difficulty_scaling = 1.8 + (1 - difficulty)
+	end
+	if difficulty > 1 then
+		manual_mining_difficulty_scaling = 1.8 - ((difficulty - 1) / 3)
+	end
+
+
 	if event == nil then
 		-- initialization/reset call
-		game.forces.player.mining_drill_productivity_bonus = game.forces.player.mining_drill_productivity_bonus + 1
-		game.forces.player.manual_mining_speed_modifier = game.forces.player.manual_mining_speed_modifier + 1
+		game.forces.player.mining_drill_productivity_bonus = 1 --fixed typo?

```

Comment:
Minin productivity starts at +100% , intentionaly. Also, it is possible that the initial mining productivity is non-zero (for example from installed mod), thus why it is not hard-written 1

Reply:
I understood that it starts at +100%, but I didn't think about the mods. Thanks!

Btw, I am making some more changes, along with providing the balance changes as tooltips in the difficulty poll.